### ISSUE TITLE
Review Instructors Handbook

### ISSUE THREAD

**Issue by maneesha** [carpentries/handbook-beta#138]
See https://carpentries-beta-handbook-preview.netlify.app/handbooks/instructors

Please consider the following when you review:

- [ ] Is all the content accurate?
- [ ] Do the links all work?
- [ ] Have you noticed any content duplicated in other places? 
- [ ] Any other feedback or suggestions

**Comment by sheraaronhurt** [comment]
@maneesha can you please reassign this to @danielle06 . She will manage the Instructors Handook

**Comment by kariljordan** [comment]
I'm reviewing the instructors handbook and here are my notes:

- The [Checkout Instructions](https://carpentries.github.io/instructor-training/checkout/index.html) link is broken.
- For off boarding, should we make it clear that emailing the team will remove them from the list, but they may still be listed as an active instructor? I recommend including more of a script in the offboarding request. What exactly should they say in this email so that we can prevent a back and forth email thread?
- Workshop Administration Team should be changed to Workshops and Instruction Team.
- Under upcoming workshops we reference "Active Instructor". Is there a place we can link to that definition in case someone doesn't know what it means?
- Under Community Calendar and Etherpad, there's a reference to Community Discussions. This should say Community Sessions, and link to the correct [Community Sessions Etherpad](https://pad.carpentries.org/community-sessions-2024).
- Other links needing updating: [Teaching and Hosting — The Carpentries Handbook documentation](https://docs.carpentries.org/topic_folders/hosts_instructors/index.html#resources-for-online-workshops), https://carpentries.org/online-workshop-recommendations/, [List of frequently asked question](https://carpentries.org/workshop_faq/) - not broken, but links to old website, [Helpers](https://docs.carpentries.org/topic_folders/hosts_instructors/hosts_instructors_checklist.html#helper-checklist), [handbook has instructions](https://docs.carpentries.org/topic_folders/for_instructors/current_instructors.html#teaching), [Core Curriculum](https://carpentries.org/workshops/#workshop-core), [Core Curricula](https://carpentries.org/workshops/#workshop-core) >>> _also, "curriculum" refers to a single plan of study; "curricula" refers to multiple plans of study_...updating the wording for consistency because they lead to the same link., [donation](https://carpentries.org/donate/), [instruction roles](https://carpentries.org/online-workshop-recommendations/#instructional-roles)
- There's reference to an updated guide for teaching [online workshop](https://docs.google.com/document/d/1Q8re6o0HO32mYyV0SgY3boFiAFxjjdSl3SKT5_WtTH8/edit). The link goes to a Google doc. Are we planning to maintain this?

**Comment by maneesha** [comment]
@danielle06 I saw you put in several PRs for the Instructor Handbook.  Can you confirm that that completes your review of the handbook? 
Can you also share your responses to Kari's feedback above? 

Thank you!

**Comment by danielle06** [comment]
@maneesha , can we keep this open for the urgent fixes? Overall, there are a few links that still will need updated but I will be discussing what to do with them at the WIT meeting Wednesday. I made a few changes to the FAQ, but because it is a bigger issue, we planned to move forward as-is and do the overhaul as a team. And now I am realising I had started to make changes to the offboarding section of the handbook but don't know that I pushed them through, so I will have to double-check that when we get our second look at things.  Thank you for all of your work on this!

@kariljordan my response to Maneesha above covers some of the issues you mention in your review. I am adding additional comments below:

- I updated the links as needed but there were some links that I still need to discuss with other people on CT
- Workshop Administration Team has been updated to Workshops and Instruction Team
- I took out "Active Instructor" and changed it to Carpentries certified Instructor and linked to the "Become an Instructor page"
- Under Community Calendar and Etherpad, there's a reference to Community Discussions. This should say Community Sessions, and link to the correct [Community Sessions Etherpad](https://pad.carpentries.org/community-sessions-2024). - _I updated the verbiage to Community Sessions. However, this is again a bigger discussion about the links. We would have to update all of the etherpad links annually in the Handbook. Checking with others on the team to decide the best way to handle this._
- Updated curriculum/curricula to correct form
- Checking with the team about the Google Doc

I will keep come back to this for my next round of fixes to make sure anything that was missed is included. Lots of work but it's coming along!
